src/services/filters.js

```diff
@@ -21,11 +22,164 @@ goog.require('ngeo');
  * @ngname ngeoScalify
  */
 ngeo.Scalify = function($filter) {
-  var number = $filter('number');
+  var numberFilter = $filter('number');
   return function(scale) {
-    var text = number(scale, 0);
+    var text = numberFilter(scale, 0);
     return text ? '1\u00a0:\u00a0' + text : '';
   };
 };
 
 ngeo.module.filter('ngeoScalify', ngeo.Scalify);
+
+
+/**
+ * Format a couple of numbers as Swiss coordinates.
+ *
+ * Example without parameters:
+ *
+ *      <p>{{[600000.1234, 200000.5678] | ngeoSwissCoordinates}}</p>
+ *      <!-- will Become 600'000, 200'001 -->
+ *      <p>{{[2600000.1234, 1200000.5678] | ngeoSwissCoordinates}}</p>
+ *      <!-- will Become 2'600'000, 1'200'001 -->
+ *
+ * Example with defined prefix, separator and suffix (you can
+ *     not use the colon symbole).
+ *
+ *      <p>{{[600000.1234, 200000.5678] | ngeoSwissCoordinates:[:; :]}}</p>
+ *      <!-- will Become [600'000; 200'001] -->
+ *      <p>{{[2600000.1234, 1200000.5678] | ngeoSwissCoordinates:[:; :]}}</p>
+ *      <!-- will Become [2'600'000; 1'200'001] -->
+ *
+ * @return {function(ol.Coordinate, string, string, string): string} A
+ *     function to format numbers into an Swiss coordinates string.
+ * @ngInject
+ * @ngdoc filter
+ * @ngname ngeoSwissCoordinates
+ */
+ngeo.SwissCoordinates = function() {
```

Comment:
Do we relay need this? for me we can do every thing with EastNorthCoordinates with a format...